scripts/update.dependencies.sh

```diff
@@ -0,0 +1,328 @@
+echo "" >> ../major.txt
+echo "shared-libs/server-checks" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/server-checks >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/task-utils" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/task-utils >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/transitions" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/transitions >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/validation" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/validation >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/calendar-interval" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/calendar-interval >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/contact-types-utils" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/contact-types-utils >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/lineage" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/lineage >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/message-utils" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/message-utils >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/phone-number" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/phone-number >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/registration-utils" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/registration-utils >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/search" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/search >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/settings" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/settings >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/tombstone-utils" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/tombstone-utils >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/translation-utils" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/translation-utils >> ../major.txt
+echo "" >> ../major.txt
+echo "shared-libs/view-map-utils" >> ../major.txt
+updates  -e bootstrap,bootstrap-daterangepicker,select2,jquery,helmet -f ./shared-libs/view-map-utils >> ../major.txt
```

Comment:
Great initiative!

I'm a little uneasy about 300+ lines of bash with quite a lot of duplication. Perhaps we can remove it from this PR and work on it separately?

A few suggestions would be...
- use node for scripting for cross-platform compatibility
- loop through directories dynamically where possible (eg: shared-libs) so it automatically includes new dirs
- provide the dependencies to exclude as a constant so we can update them in one place (we want to be continuously working on reducing the list so we can stay up to date)

Reply:
Removed in 2692dc390 and added in #7510 - thanks for the feedback!